OptimoveSDK/optimove-sdk/src/main/java/com/optimove/sdk/optimove_sdk/optimobile/OptimobileConfig.java

```diff
@@ -0,0 +1,366 @@
+package com.optimove.sdk.optimove_sdk.optimobile;
+
+import android.util.Base64;
+
+import androidx.annotation.DrawableRes;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.optimove.sdk.optimove_sdk.R;
+import com.optimove.sdk.optimove_sdk.main.tools.opti_logger.LogLevel;
+
+import org.json.JSONObject;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+
+/**
+ * Represents the configuration for the Optimobile client
+ */
+public final class OptimobileConfig {
```

Comment:
Since this features prominently in the integration guide worth renaming this to OptimoveConfig ?

Reply:
Is this config specific to Optimobile tho, do we have another one that's Optimove?

If this is the generic SDK config, agree on renaming, but also then probably should lift package up to base rather than sit in Optimobile?

Reply:
Our sample apps suggest that integration looks like

```
Optimove.initialize(this.getApplication(), new OptimobileConfig.Builder([string],[string]).build());
```

So this does seem to be the only config?

Reply:
ah ok, then yep, seems like maybe need to lift up a package & rename (Y)